pkg/consts/kongplugininstallation.go

```diff
@@ -8,4 +8,8 @@ const (
 	// AnnotationKongPluginInstallationName is the annotation key used to store the name of the KongPluginInstallation
```

Comment:
```suggestion
	// AnnotationKongPluginInstallationMappedKongPluginInstallation is the annotation key used to store the name of the KongPluginInstallation
```

Reply:
I'm hoping you think about something like I have know 😉